recipe/meta.yaml

```diff
@@ -18,11 +18,10 @@ build:
   string: py{{ CONDA_PY }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ torchani_proc_type }}                                                  # [cuda_compiler_version == "None"]
   string: py{{ CONDA_PY }}cuda{{ cuda_compiler_version|replace('.', '') }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}_{{ torchani_proc_type }}  # [cuda_compiler_version != "None"]
   script:
-    # cuda 10.2 does not have cub, which need to be manually downloaded
-    # - sudo yum install wget unzip -y                                                  # [cuda_compiler_version == '10.2']
-    # However as the current master version of cub has conflict with cuda 10.2, cuaev will not support cuda 10.2 for now.
-    - "{{ PYTHON }} -m pip install . -vvv --no-deps"                                    # [cuda_compiler_version in ("None", '10.2')]
-    - "{{ PYTHON }} -m pip install . --global-option='--cuaev-all-sms' -vvv --no-deps"  # [cuda_compiler_version not in ("None", '10.2')]
+    # cuda 10.2 does not have cub, which need to be manually downloaded in setup.py
+    - sudo yum install wget unzip -y                                                    # [cuda_compiler_version == '10.2']
```

Comment:
In the deps `build` section you can add `unzip` and `wget`: https://github.com/conda-forge/unzip-feedstock and https://github.com/conda-forge/wget-feedstock

Reply:
I didn't know this, let me try!

Reply:
More generally and FYI, conda-forge does not only contain python packages but also other languages and very often widely common tools such as `wget`, `unzip`, `git`, and many others. Usually and whenever possible it's best to stick to conda forge for any dependencies instead of relying on `yum`.

Reply:
Gotcha